docs/usage/ruby.md

```diff
@@ -1,29 +1,23 @@
 ---
-title: Ruby Bundler Support
+title: Ruby Bundler support
 description: Ruby Bundler support in Renovate
 ---
 
-# Automated Dependency Updates for Ruby Bundler Dependencies
+# Automated dependency updates for Ruby Bundler dependencies
 
 Renovate supports upgrading dependencies in Bundler's `Gemfile`s and their accompanying `Gemfile.lock` files.
 Support is considered "alpha" stage until there have been some more real-world tests.
```

Comment:
```suggestion

```

Can we maybe remove this sentence? What is the status of Ruby support for Renovate right now?

Reply:
Probably. We have a `releaseStatus` property which can be `alpha` or `beta`. Bundler is neither of those, and hasn't seen above average number of issues or bug fixes IMO.

Reply:
Yes, let's remove it.l